lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Co takhle to doplnit o scatter plot s vizualizací různých tříd a kombinací vstupních parametrů? Tady by to mohlo být ještě reálné a umožnit si ten problém lépe představit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>